corva/models/stream.py

```diff
@@ -28,24 +23,44 @@ class Record(BaseEventData):
     data: RecordData
 
 
-class AppMetadata(BaseEventData):
+class AppMetadata(CorvaBaseModel):
     app_connection_id: int
     app_version: Optional[int] = None
```

Comment:
I think we can make this not Optional. We want this all the time, and anybody who is using the SDK will also be using Dev Center or otherwise deploying with Packager, which should help for this.

Reply:
Asked Ville about this field, here is what he said

> I think it should be optional since old apps might not have it

Reply:
Old apps won't be using the Python SDK until they are updated though.

Reply:
made the field required